src/app/project/style.scss

```diff
@@ -25,6 +25,31 @@ $project-card-width: 330px;
   margin: 20px auto;
 }
 
+.placeholder {
+  height: 75vh;
+
+  & h2 {
+    font-weight: 500;
+    margin: 30px 0;
+  }
+
+  & .placeholder-text {
+    & p {
+      margin: 0 0 10px;
+    }
+  }
+
+  .mat-flat-button {
```

Comment:
Why do we need to adjust button size? It should be same as regular "Add Project" button.